### PR TITLE
Add new icons

### PR DESCRIPTION
**Pull request by WiIIiam278**
image

### CODE REVIEW

**Comment by WiIIiam278** [comment]
oh, my branch was out of date, hence the weird first commit.

File: SerialLoops.sln [hunk 1]
```diff
@@ -10,28 +10,29 @@ EndProject
...
-        {CC729174-57CD-499A-B6E7-1FA6F9CE255F}.Release|Any CPU.ActiveCfg = Release|Any CPU
-        {CC729174-57CD-499A-B6E7-1FA6F9CE255F}.Release|Any CPU.Build.0 = Release|Any CPU
-    EndGlobalSection
-    GlobalSection(SolutionProperties) = preSolution
-        HideSolutionNode = FALSE
-    EndGlobalSection
-    GlobalSection(ExtensibilityGlobals) = postSolution
-        SolutionGuid = {A7259B55-BEEA-4ECC-AC0A-85CC8FA1BD53}
-    EndGlobalSection
+	GlobalSection(SolutionConfigurationPlatforms) = preSolution
```
Comment (jonko0493) [inline SerialLoops.sln:13]: Looks like these became tabs again -- might want to check your editor's config and make sure it's using spaces

**Review by jonko0493 — APPROVED** [review]

File: SerialLoops.sln (hunk 1 above)
Comment (jonko0493) [reply SerialLoops.sln:13]: nvm this got fixed lol

Comment (WiIIiam278) [reply SerialLoops.sln:13]: not sure what's up with that lol, but also I'm not manually editing the .sln file so /shrug

**Pull request merged**